Documentation/gettingstarted/memcached.rst

```diff
@@ -0,0 +1,324 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+**********************************
+Getting Started Securing Memcached
+**********************************
+
+This document serves as an introduction to using Cilium to enforce memcached-aware
+security policies. It walks through a single-node
+Cilium environment running on your machine. It is designed to take 15-30
+minutes.
+
+.. include:: gsg_intro.rst
+
+**NOTE:** memcached-aware policy support is still "experimental".  It is not yet ready for
```

Comment:
We use the terminology beta instead of experimental. I've just fixed this up for the Cassandra GSG as well.